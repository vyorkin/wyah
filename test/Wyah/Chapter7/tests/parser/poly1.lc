let f = I (I I) (I 3);
