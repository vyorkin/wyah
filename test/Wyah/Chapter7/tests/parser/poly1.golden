Program [Decl "f" (EApp (EApp (EVar (Var {unVar = "I"})) (EApp (EVar (Var {unVar = "I"})) (EVar (Var {unVar = "I"})))) (EApp (EVar (Var {unVar = "I"})) (ELit (LInt 3))))]